### DISCUSSION TITLE
I would prefer to just see my organizations

### DISCUSSION THREAD

**Discussion by hoani** [community/community#15096] [Other Feature Feedback, Questions, & Ideas]
Rather than a feed which pushes out information I don't care about (github isn't social media after all); I would prefer to have the various organizations I'm a part of, and potentially the repositories I have most recently contributed to show up on the landing page.

It's pretty frustrating when you go to github with the intention of looking for a repository you're working on, and you have to click through various links to find something you were working on recently

**Comment by jfullstackdev** [comment]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
